WIP: Listing filters

NTWEXO opened the pull request:
<!--
Thank you for contributing to Shopware! Please fill out this description template to help us to process your pull request.

Please make sure to fulfil our contribution guideline (https://docs.shopware.com/en/shopware-platform-dev-en/contribution/contribution-guideline?category=shopware-platform-dev-en/contribution).

Do your changes need to be mentioned in the documentation?
Add notes on your change right now in the documentation files in /src/Docs/Resources and add them to the pull request as well. 
-->

### 1. Why is this change necessary?


### 2. What does this change do, exactly?


### 3. Describe each step to reproduce the issue or behaviour.


### 4. Please link to the relevant issues (if any).


### 5. Checklist

- [ ] I have written tests and verified that they fail without my change
- [ ] I have squashed any insignificant commits
- [ ] I have written or adjusted the documentation according to my changes
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [ ] I have read the contribution requirements and fulfil them.

shyim commented on engine/Shopware/Core/sAdmin.php:
```diff
@@ -2506,6 +2506,13 @@ public function sGetDispatchBasket($countryID = null, $paymentID = null, $stateI
         $userId = $this->session->offsetGet('sUserId');
+
+        $sql = $this->eventManager->filter(
+            'Shopware_Modules_Admin_GetDispatchBasket_FilterResult',
```
Should the event name not end with "_FilterSql"? FilterResult is mostly used for result of queries

mark-newman replied:
Yes, good idea. I've made the change.

mark-newman commented on engine/Shopware/Core/sAdmin.php:
```diff
@@ -2506,6 +2506,13 @@ public function sGetDispatchBasket($countryID = null, $paymentID = null, $stateI
+        $sql = $this->eventManager->filter(
+            'Shopware_Modules_Admin_GetDispatchBasket_FilterSql',
+            $sql,
+            ['subject' => $this, 'id' => $userId]
```
Done

mark-newman commented on engine/Shopware/Core/sAdmin.php:
```diff
@@ -2726,6 +2733,13 @@ public function sGetPremiumDispatches($countryID = null, $paymentID = null, $sta
+        $sql = $this->eventManager->filter(
+            'Shopware_Modules_Admin_GetPremiumDispatches_FilterSql',
+            $sql,
+            ['subject' => $this, 'id' => $userId]
```
Done

shyim commented:
Hey @mark-newman ,
we have changed these methods to use a QueryBuilder and provided there new events to modifiy the QueryBuilder with commit https://github.com/shopware/shopware/commit/eec8927194a374cd9ac534adee60d9b2aa5bd327.
This should solve your issue, but thanks for your Contribution :blue_heart: :tada:

JoshuaBehrens committed:
Support keys in collection constructor

JoshuaBehrens committed:
Amend behaviour in entity collection constructor

soebbing committed:
NEXT-7909 - Add some interfaces

Signed-off-by: Hendrik Söbbing <hendrik@soebbing.de>

Closes https://github.com/shopware/platform/pull/556

JoshuaBehrens committed:
Add test to verifiy keeping keys in collections

JoshuaBehrens committed:
Stop wrapping single filter in multi filter

JoshuaBehrens committed:
Clone criteria to prevent internal changes

windaishi committed:
Json-serialize dates with milliseconds

windaishi committed:
NEXT-9608 - Change JSON date format for classes with alternative implementations of jsonSerialize

Closes https://github.com/shopware/platform/pull/623

JoshuaBehrens committed:
NEXT-7704 - Add checks for criteria consistency after search calls

Closes https://github.com/shopware/platform/pull/587

jkrzefski committed:
NEXT-9624 - Add new commands system:config:get and system:config:set

Closes https://github.com/shopware/platform/pull/657

JoshuaBehrens committed:
NEXT-7315 - Use custom way to add in constructor in price collection

Closes https://github.com/shopware/platform/pull/471

jkrzefski committed:
NEXT-9209 - Not call actions of business event when its propagation was stopped

fixes https://github.com/shopware/platform/pull/720

troxxs committed:
NEXT-9257 - stringify theme text and textarea values (not only media)

fixes https://github.com/shopware/platform/pull/722

GitEvil committed:
NEXT-8334 - Make notification titles consistent and fix e2e tests

Moomboh committed:
NEXT-9269 - Add new rule CustomerTagRule and tags for customers
closes https://github.com/shopware/platform/pull/1003/

vttn committed:
NEXT-9067 - Stop email being send out if MailBeforeValidateEvent is stop propagation

fixes https://github.com/shopware/platform/pull/1027

lernhart committed:
NEXT-9201 - changed propagation check to work with all kinds of event object types

Commit:
NEXT-7043 - display product variations in administration

lernhart committed:
NEXT-9257 - added strict to in_array check and removed text from being stringified

Commit:
NEXT-5047 - Add proper variant custom field inheritance

Haberkamp committed:
NEXT-9270 - fix sorting products only via default currency

philipreinken committed:
NEXT-9203 - Add essential characteristics to line item data

leichteckig committed:
NTR - Add e2e test for validation after auto-update

leichteckig committed:
NTR - Use before hook in storefront e2e test

taltholtmann committed:
NEXT-9225 -  Add product custom field set association

tobiasberge committed:
NEXT-7043 - Add multiLine option for data-grid columns

Commit:
NEXT-9619 - add namespace variable to sw_icon
fixes #1071
closes #1077

hungmac-sw committed:
NEXT-9350: Fixed filter dropdowns are cut off

susannemoog committed:
NEXT-6182 Fix calculation of shipping costs

leichteckig committed:
NEXT-7043 - Adjust variant specification component to new screendesign

philipreinken committed:
NEXT-8230 - Show essential characteristics in all checkout-related product listings

Commit:
NEXT-9491 - Fixed changelogs.
Fixes https://github.com/shopware/platform/pull/1085.

Commit:
NEXT-9205 - add context token required parameter

Commit:
NEXT-9205 - add context token required Annotation

keulinho committed:
SAAS-401 - Add app snippet docs

Commit:
NEXT-9278 - add logging and deprecation notice for exception

tobiasberge committed:
NEXT-7043 - Use flexbox for single select value and input

leichteckig committed:
NEXT-7043 - Hide additional text if variant is grouped

Commit:
NEXT-9599 - Replaced "declare(strict_types = 1)" to "declare(strict_types=1)" the documentation.
Fixes https://github.com/shopware/platform/pull/1103.

tobiasberge committed:
Merge branch 'next-8868/adjust-variant-charactieristics-to-screendesign' into 'master'

NEXT-8868 - Implement new storefront screendesign for variant characteristics

See merge request shopware/6/product/platform!2549

OliverSkroblin committed:
Merge branch 'next-9338/optimize-es-index' into 'master'

NEXT-9338 - Change es indexing behavior, that the product update will be applied immediately

See merge request shopware/6/product/platform!2609

OliverSkroblin committed:
Merge branch 'next-9542/fix-route-deprecation' into 'master'

NEXT-9542 - Route deprecations

See merge request shopware/6/product/platform!2615

shyim committed:
NEXT-9595 - Fix payment redirect url in api response

Haberkamp committed:
NEXT-7025 - only save edited custom fields

pweyck committed:
Merge branch 'next-9599/improved-documentation-consistency' into 'master'

NEXT-9599 - Replaced "declare(strict_types = 1)" to "declare(strict_types=1)" the documentation.

See merge request shopware/6/product/platform!2628

pweyck committed:
Merge branch 'next-9566/fix-ide-twig-json-for-storefront' into 'master'

NEXT-9566 - fix path rename of "@Storefront/storefront" also for...

See merge request shopware/6/product/platform!2618

shyim committed:
NEXT-6728 - Fix storefront watcher

GitEvil committed:
NEXT-7521 - Fix VAT calculation snippets

OliverSkroblin committed:
Merge branch 'next-9595/fix-payment-response-for-json' into 'master'

NEXT-9595 - Fix payment redirect url in api response

See merge request shopware/6/product/platform!2630

OliverSkroblin committed:
Merge branch 'next-9203/add-essential-characteristics-to-lineitem' into 'master'

NEXT-9203 - Add essential characteristics to line item data

See merge request shopware/6/product/platform!2581

OliverSkroblin committed:
Merge branch 'next-7521-general-snippet-fixes' into 'master'

NEXT-7521 - Fix VAT calculation snippets

See merge request shopware/6/product/platform!2632

keulinho committed:
NTR - Remove github mirror in pipeline

Repository is mirrored automatically by gitlab,
the step in our pipeline is not neeeded anymore

leichteckig committed:
NEXT-7043 - Rename sw-product-variation to sw-product-variant-info

leichteckig committed:
NEXT-7043 - Add variant characteristics to offcanvas cart

windaishi committed:
NEXT-4521 - Add remove plugin data uninstall modal

Closes https://github.com/shopware/platform/pull/1177

tobiasberge committed:
Merge branch 'next-6728/fix-watcher' into 'master'

NEXT-6728 - Fix storefront watcher

See merge request shopware/6/product/platform!2631

pweyck committed:
Merge branch 'next-9608/pickware/serialize-json-dates-with-milliseconds' into 'master'

NEXT-9608/pickware/serialize json dates with milliseconds

See merge request shopware/6/product/platform!2635

leichteckig committed:
NEXT-7043 - Simplify additional text usage in sw-select

shyim committed:
NEXT-8818 - Migrate admin order to store-api

pweyck committed:
NEXT-7315 - Implement missing collection::set methods

- fix tests

JoshuaBehrens committed:
NEXT-9743 - Add is merged flag to line item added event.
Fixes https://github.com/shopware/platform/pull/1115.

Commit:
NEXT-9619 - update changelog

Commit:
NEXT-9634 - fix checkbox (toggle group) event propagation on mobile devices

fixes https://github.com/shopware/platform/pull/1116

tobiasberge committed:
Merge branch 'next-8818/admin-order-store-api' into 'master'

NEXT-8818 - Migrate admin order to store-api

See merge request shopware/6/product/platform!2627

pweyck committed:
Merge branch 'ntr/remove-github-mirror-in-pipeline' into 'master'

NTR - Remove github mirror in pipeline

See merge request shopware/6/product/platform!2633

taltholtmann committed:
Merge branch 'next-9428/pagination-cannot-jump-to-fourth-page' into 'master'

NEXT-9428 - pagination cannot jump to fourth page

See merge request shopware/6/product/platform!2602

shyim committed:
NEXT-8125 - Remove legacy password on password reset

tobiasberge committed:
Merge branch 'next-9350/fixed-filter-dropdowns-are-cut-off' into 'master'

NEXT-9350: Fixed filter dropdowns are cut off

See merge request shopware/6/product/platform!2563

pweyck committed:
NEXT-9627 - Skip flaky plugin integration tests

OliverSkroblin committed:
Merge branch 'next-7704/bugfix/change-of-criteria' into 'master'

NEXT-7704/bugfix/change of criteria

See merge request shopware/6/product/platform!2641

mitelg committed:
Merge branch 'next-7909/add-interfaces' into 'master'

NEXT-7909 - Add some interfaces

See merge request shopware/6/product/platform!2638

OliverSkroblin committed:
Merge branch 'next-7315/support-keys-in-collection-ctor' into 'master'

NEXT-7315/support keys in collection ctor

See merge request shopware/6/product/platform!2639

OliverSkroblin committed:
Merge branch 'next-9627/skip-plugin-integration-tests' into 'master'

NEXT-9627 - Skip flaky plugin integration tests

See merge request shopware/6/product/platform!2653

OliverSkroblin committed:
Merge branch 'next-9619/add-namespace-variable-to-sw-icon' into 'master'

NEXT-9619 - add namespace variable to sw icon

See merge request shopware/6/product/platform!2645

tobiasberge committed:
Merge branch 'next-7043/add-product-variation-in-administration' into 'master'

NEXT-7043 - Add product variation in administration

See merge request shopware/6/product/platform!2369

pweyck committed:
Merge branch 'next-9205/add-needs-context' into 'master'

NEXT-9205 - add context token required parameter

See merge request shopware/6/product/platform!2603

mitelg committed:
Merge branch 'next-9624/add-system-config-commands' into 'master'

NEXT-9624 - Add new commands system:config:get and system:config:set

See merge request shopware/6/product/platform!2651

pweyck committed:
Merge branch 'master' into 'next-9491/change-shipping-free-item-behavior'

# Conflicts:
#   CHANGELOG-6.3.md

pweyck committed:
Merge branch 'next-9491/change-shipping-free-item-behavior' into 'master'

NEXT-9491 - change shipping free item behavior

See merge request shopware/6/product/platform!2590

leichteckig committed:
NEXT-9600 - Remove feature flag for additional text

tobiasberge committed:
Merge branch 'next-9634/fix-checkbox-event-propagation-on-mobile-devices' into 'master'

NEXT-9634 - fix checkbox (toggle group) event propagation on mobile devices

See merge request shopware/6/product/platform!2655

shyim committed:
Update 20-installation.md

taltholtmann committed:
Merge branch 'next-9257/stringify-theme-text-and-and-textarea' into 'master'

NEXT-9257 - stringify theme textarea values (not only media)

See merge request shopware/6/product/platform!2469

tobiasberge committed:
Merge branch 'ntr/change-install-to-http' into 'master'

Update 20-installation.md

See merge request shopware/6/product/platform!2659

leichteckig committed:
NEXT-9600 - Add Jest tests for sw-product-variant-info

shyim committed:
NEXT-9609 - Changed NavigationRoute to allow sending custom parameters in POST

sydinh committed:
NTR - Add SCSS mixins

Commit:
NEXT-9269 - Add CustomerTag indexer

OliverSkroblin committed:
Merge branch 'next-8125/reset-legacy-password-on-reset' into 'master'

NEXT-8125 - Remove legacy password on password reset

See merge request shopware/6/product/platform!2652

leichteckig committed:
NEXT-9600 - Implement epic test feedback for additional text

keulinho committed:
NEXT-9668 - Add PK to mail_template_sales_channel table

GitEvil committed:
NEXT-7521 - storefront snippet fixes and corresponding e2e fixes

taltholtmann committed:
Merge branch 'ntr/add-scss-mixins' into 'master'

NTR - Add SCSS mixins

See merge request shopware/6/product/platform!2669

Commit:
NTR - Added headers to cors listener allowed list

pweyck committed:
Merge branch 'next-9668/add-pk-to-mail-template-sales-channel-table' into 'master'

NEXT-9668 - Add PK to mail_template_sales_channel table

See merge request shopware/6/product/platform!2672

leichteckig committed:
Merge branch 'next-8230/show-essential-characteristics-in-storefront' into 'master'

NEXT-8230 - Show essential characteristics in all checkout-related product listings

See merge request shopware/6/product/platform!2647

mkrafeld committed:
SAAS-504 - add app template documentation

tobiasberge committed:
Merge branch 'next-9600/additional-text-feedback' into 'master'

Next 9600/additional text feedback

See merge request shopware/6/product/platform!2678

pweyck committed:
Merge branch 'ntr/added-seo-urls-header-to-cors-allowed' into 'master'

NTR - Add seo urls header to cors allowed

See merge request shopware/6/product/platform!2675

powolnymarcel committed:
NEXT-9750 - Add PK to various migrations

fixes #1141, fixes #1142, fixes #1143, fixes #1144, fixes #1145, fixes #1146, fixes #1147, fixes #1148, fixes #1149, fixes #1150, fixes #1151, fixes #1152, fixes #1153

taltholtmann committed:
NEXT-9549 - Inheritance for advanced prices and properties

keulinho committed:
SAAS-1239 - Add app privacy terms docs

philipreinken committed:
NEXT-8230 - Prevent missing reference price display, with inactive essential characteristics feature flag

tobiasberge committed:
Merge branch 'next-8230/include-feature-flag' into 'master'

NEXT-8230 - Prevent missing reference price display, with inactive essential...

See merge request shopware/6/product/platform!2691

taltholtmann committed:
NEXT-9225 - Add feature flag

Commit:
Merge branch 'next-9609/fix-post-tree-navigation-route' into 'master'

NEXT-9609 - Changed NavigationRoute to allow sending custom parameters in POST

See merge request shopware/6/product/platform!2668

pweyck committed:
Merge branch 'next-9269/add-customer-tag-rule' into 'master'

NEXT-9269 - Add new rule CustomerTagRule and tags for customers

See merge request shopware/6/product/platform!2479

lernhart committed:
NEXT-9750 - added changelog, imported DBALException in missing migration and set --sql-require-primary-key=ON on gitlab ci

leichteckig committed:
NEXT-9600 - Adjust storefront product box styling to screendesign

lernhart committed:
NEXT-9067 - added changelog

pweyck committed:
Merge branch 'next-9209/fix-event-stop-propagation' into 'master'

NEXT-9209 - Not call actions of business event when its propagation was stopped

See merge request shopware/6/product/platform!2450

Commit:
NEXT-9743 - Added changelog entry & fixed tests

pweyck committed:
Merge branch '6.2'

SebastianFranze committed:
Merge branch 'saas-1239/add-app-privacy-terms-doc' into 'master'

SAAS-1239 - Add app privacy terms docs

See merge request shopware/6/product/platform!2688

mitelg committed:
NEXT-9269 - Fix namespace of CustomerIndexerEvent

Commit:
Merge branch 'next-9269/fix-namespace' into 'master'

NEXT-9269 - Fix namespace of CustomerIndexerEvent

See merge request shopware/6/product/platform!2699

tobiasberge committed:
NTR - fix password validation for updating users

mitelg committed:
NTR - Remove empty files

mitelg committed:
NTR - Fix code style

mitelg committed:
NTR - Fix visibility of constants

pweyck committed:
Merge branch 'ntr/clean-up-code' into 'master'

Ntr/clean up code

See merge request shopware/6/product/platform!2700

shyim committed:
NEXT-9767 - Remove usage of StoreHttpClient

leichteckig committed:
NEXT-9600 - Update email template order confirmation

shyim committed:
NEXT-9766 - Set default asset package

pweyck committed:
Merge branch 'next-9750/add-pk-to-various-migrations' into 'master'

NEXT-9750 - add pk to various migrations

See merge request shopware/6/product/platform!2695

Commit:
Merge branch 'next-9600/remove-feature-flag' into 'master'

NEXT-9600 - Remove feature flag for additional text

See merge request shopware/6/product/platform!2656

leichteckig committed:
NEXT-9600 - Minor epic test style fixes

taltholtmann committed:
NEXT-9549 - Fix inheritance label

tobiasberge committed:
Merge branch 'next-9600/further-feedback' into 'master'

NEXT-9600 - Minor epic test style fixes

See merge request shopware/6/product/platform!2704

OliverSkroblin committed:
Merge branch 'next-9766/set-default-symfony-asset-page' into 'master'

NEXT-9766 - Set default asset package

See merge request shopware/6/product/platform!2702

OliverSkroblin committed:
Merge branch 'next-9767/remove-cart-store-client-usage' into 'master'

NEXT-9767 - Remove usage of StoreHttpClient

See merge request shopware/6/product/platform!2701

OliverSkroblin committed:
Merge branch 'next-9743/added-is-merged-to-line-item-event' into 'master'

NEXT-9743 - Add is merged flag to line item added event.

See merge request shopware/6/product/platform!2697

shyim committed:
NEXT-9606 - Mark price fields as read protected in the Store/Sales-Channel API

pweyck committed:
NEXT-9790 - Skip translations for unknown languages

OliverSkroblin committed:
Merge branch 'next-9606/mark-price-fields-as-read-protected' into 'master'

NEXT-9606 - Mark price fields as read protected in the Store/Sales-Channel API

See merge request shopware/6/product/platform!2706

Commit:
NEXT-9792 - Fix sw_sanitize throws on null options

shyim committed:
NTR - Fix MySQL 8 Build

philipreinken committed:
NTR - Remove migration for essential characteristics

SebastianFranze committed:
Merge branch 'next-9792/fix-sw-sanitize-throws-when-options-null' into 'master'

NEXT-9792 - Fix sw_sanitize throws on null options

See merge request shopware/6/product/platform!2708

tobiasberge committed:
Merge branch 'next-9270/fix-sorting-products-only-via-default-currency' into 'master'

NEXT-9270 - fix sorting products only via default currency

See merge request shopware/6/product/platform!2614

OliverSkroblin committed:
Merge branch 'ntr/remove-essential-characteristics-migration' into 'master'

NTR - Remove migration for essential characteristics

See merge request shopware/6/product/platform!2710

OliverSkroblin committed:
Merge branch 'next-9790/skip-translations-for-unknown-languages' into 'master'

NEXT-9790 - Skip translations for unknown languages

See merge request shopware/6/product/platform!2707

tobiasberge committed:
Merge branch 'NEXT-8334/consistent-notification-titles' into 'master'

NEXT-8334 - Make notification title snippets consistently use the global namespace

Closes NEXT-8334

See merge request shopware/6/product/platform!2151

pweyck committed:
Merge branch 'ntr/fix-mysql8-build' into 'master'

NTR - Fix MySQL 8 Build

See merge request shopware/6/product/platform!2709

OliverSkroblin committed:
NTR - Remove v6.3.0 deprecation in login service

shyim committed:
NTR - Fix github ci

tobiasberge committed:
Merge branch 'ntr/remove-deprecation' into 'master'

NTR - Remove v6.3.0 deprecation in login service

See merge request shopware/6/product/platform!2712

leichteckig committed:
NTR - Use mochawesome reports

pweyck committed:
Merge branch 'next-9549/fix-inheritance-and-add-custom-field-selection' into 'master'

Next 9549/fix inheritance and add custom field selection

See merge request shopware/6/product/platform!2690

pweyck committed:
Merge branch 'next-9067/stop-mails-on-stopped-event-propagation' into 'master'

NEXT-9067 - stop mails on stopped event propagation

See merge request shopware/6/product/platform!2696

Commit:
Merge branch master

pweyck committed:
Merge branch 'ntr/fix-github-ci' into 'master'

NTR - Fix github ci

See merge request shopware/6/product/platform!2713

pweyck committed:
Merge branch 'next-9484/merge-6.2' into 'master'

NEXT-9484/merge 6.2

Closes NEXT-9176

See merge request shopware/6/product/platform!2698

pweyck committed:
Merge branch 'next-7521/general-snippet-fixes' into 'master'

NEXT-7521 - general snippet fixes

See merge request shopware/6/product/platform!2673

keulinho committed:
NEXT-9753 - Lazy load route collection in aopen api loader

pweyck committed:
Merge branch 'saas-401/add-app-snippet-docs' into '6.3.0.0'

SAAS-401 - Add app snippet docs

See merge request shopware/6/product/platform!2607

Commit:
NTR - fix e2e test sw-settings-user

pweyck committed:
Merge branch 'next-9753/lazy-load-route-collection' into '6.3.0.0'

NEXT-9753 - Lazy load route collection in aopen api loader

See merge request shopware/6/product/platform!2715

pweyck committed:
NEXT-9800 - Set version to 6.3.0.0

pweyck committed:
Merge branch 'ntr/fix-e2e-sw-settings-user' into '6.3.0.0'

NTR - fix e2e test sw-settings-user

See merge request shopware/6/product/platform!2717

taltholtmann committed:
NEXT-9807 - Increase api version

pweyck committed:
Merge branch 'next-9800/set-version' into '6.3.0.0'

NEXT-9800/set version

See merge request shopware/6/product/platform!2721

GitEvil committed:
NEXT-7521 - fix typos

pweyck committed:
Merge branch 'next-7521/fix-typos' into '6.3.0.0'

NEXT-7521 - fix typos

See merge request shopware/6/product/platform!2724

pweyck committed:
NTR - Fix update

OliverSkroblin committed:
Merge branch 'next-9816/fix-update' into '6.3.0.0'

NEXT-9816/fix update

See merge request shopware/6/product/platform!2725

Commit:
NEXT-9817 - remove deprecations of shipping method price for currency price and fix tests

Commit:
NEXT-9822 - change number of digits to four in sw-version

Commit:
NEXT-9854 - fix reportValidity polyfill missing
fixes #1170

pweyck committed:
Merge branch 'next-9822/change-number-of-digits-to-four-in-sw-version' into '6.3.0.0'

NEXT-9822 - change number of digits to four in sw-version

See merge request shopware/6/product/platform!2728

shyim committed:
NEXT-9807 - Increase api version in docs

pweyck committed:
Merge branch 'next-9807/increase-api-version' into '6.3.0.0'

Next 9807/increase api version

See merge request shopware/6/product/platform!2722

pweyck committed:
NTR - Fix installer e2e

OliverSkroblin committed:
NEXT-9814 - Fix user access key handling in administration

pweyck committed:
NTR - Log web server log into separate log file to avoid size limit in e2e

- add logs as build artifacts to ease debugging

pweyck committed:
Merge branch 'ntr/fix-installer-e2e' into '6.3.0.0'

NTR - Fix installer e2e

See merge request shopware/6/product/platform!2729

shyim committed:
NEXT-9834 - Remove double slash in Vue asset filter

leichteckig committed:
NEXT-1810 - Fix product sorting e2e test

Commit:
NEXT-9591 - fix missing link to profile in sw-admin-menu

taltholtmann committed:
NEXT-9807 - Fix api version in administration services

pweyck committed:
Merge branch 'next-9834/fix-asset-path-admin' into '6.3.0.0'

NEXT-9834 - Remove double slash in Vue asset filter

See merge request shopware/6/product/platform!2733

pweyck committed:
Merge branch 'next-9591/fix-missing-link-to-profile' into '6.3.0.0'

NEXT-9591 - fix missing link to profile in sw-admin-menu

See merge request shopware/6/product/platform!2737

pweyck committed:
Merge branch 'next-1810/fix-sorting-test-e2e' into '6.3.0.0'

NEXT-1810 - Fix product sorting e2e test

See merge request shopware/6/product/platform!2734

tobiasberge committed:
NEXT-7025 - Adjust CHANGELOG-6.3.md

pantrtxp committed:
NEXT-9810 - Fix Changelog for 6.3

pweyck committed:
Merge branch 'next-9854/fix-reportValidity-polyfill-missing' into '6.3.0.0'

NEXT-9854 - fix reportValidity polyfill missing

See merge request shopware/6/product/platform!2739

pweyck committed:
Merge branch 'ntr/change-stdout-to-log-file' into '6.3.0.0'

Store web server output in separate log file

See merge request shopware/6/product/platform!2731

taltholtmann committed:
Merge branch 'NEXT-9807/api-version-fix-in-administration' into '6.3.0.0'

Resolve NEXT-9807 "/api version fix in administration"

See merge request shopware/6/product/platform!2736

OliverSkroblin committed:
Merge branch 'next-9814/fix-admin-access-keys-2' into '6.3.0.0'

NEXT-9814 - Fix user access key handling in administration

See merge request shopware/6/product/platform!2735

tobiasberge committed:
NEXT-7025 - Return Promises for better extensibility

pweyck committed:
Merge branch 'next-4521/plugin-uninstall-prompt' into '6.3.0.0'

NEXT-4521/plugin uninstall prompt

See merge request shopware/6/product/platform!2742

pweyck committed:
Merge branch 'next-9810/fix-changelog' into '6.3.0.0'

NEXT-9810 - fix changelog

See merge request shopware/6/product/platform!2741

tobiasberge committed:
NEXT-7025 - Remove base tag from E2E test

tobiasberge committed:
NEXT-7025 - Fix comment

tobiasberge committed:
NEXT-7025 - Fix typo

keulinho committed:
NTR - Include major version in kernel fallback version

pweyck committed:
Merge branch 'next-9817/remove-trigger-shipping-price' into '6.3.0.0'

NEXT-9817 - remove trigger for deprecation of shipping method price for...

See merge request shopware/6/product/platform!2727

pweyck committed:
Merge branch 'next-7025/custom-field-fixes' into '6.3.0.0'

NEXT-7025 - Custom field fixes

See merge request shopware/6/product/platform!2718

pweyck committed:
Merge branch 'ntr/include-major-in-fallbacl-version' into '6.3.0.0'

NTR - Include major version in kernel fallback version

See merge request shopware/6/product/platform!2745

Commit:
NEXT-9817 - fix currencyid prefix in currency_price json

Hero10001 committed:
SAAS-504 - Moved and extended the app template docs

pweyck committed:
Merge branch 'next-9817/fix-currency-prefix-json' into '6.3.0.0'

NEXT-9817 - fix currencyid prefix in currency_price json

See merge request shopware/6/product/platform!2748

leichteckig committed:
NEXT-1810 - Use cypress workaround for chrome connection issue

leichteckig committed:
NEXT-1810 - Skip sort-listing e2e test temporarily

taltholtmann committed:
NEXT-9670 - Fix cloning of cloned products

OliverSkroblin committed:
Add check for listing price field serializer

pweyck committed:
Merge branch 'ntr/improve-serializer' into '6.3.0.0'

Add check for listing price field serializer

See merge request shopware/6/product/platform!2751

leduc92 committed:
NEXT-9886 Remove Google shopping sales channel type

OliverSkroblin committed:
Merge branch 'next-9886/remove-google-shopping-sales-channel-type' into '6.3.0.0'

NEXT-9886 Remove Google shopping sales channel type

See merge request shopware/6/product/platform!2756

pweyck committed:
Merge branch 'next-1810/use-workaround-for-chrome-econnreset-issue' into '6.3.0.0'

NEXT-1810 - use workaround for chrome econnreset issue

See merge request shopware/6/product/platform!2750

pweyck committed:
Merge branch 'next-9670/fix-duplicate-products' into '6.3.0.0'

NEXT-9670 - Fix cloning of cloned products

See merge request shopware/6/product/platform!2753

keulinho committed:
NEXT-9598 - Always provide theme translations in default language

vienthuong committed:
NEXT-9886 - Remove Google Shopping Related Tables

pweyck committed:
Merge branch 'next-9598/provide-theme-translations-in-default-language' into '6.3.0.0'

NEXT-9598 - Always provide theme translations in default language

See merge request shopware/6/product/platform!2757

keulinho committed:
SAAS-504 - Link github repos

philipreinken committed:
NEXT-9899 - Remove a duplicate condition from order confirmation mail template

pweyck committed:
Merge branch 'SAAS-504/add-app-template-documentation' into '6.3.0.0'

SAAS-504 - add app template documentation

See merge request shopware/6/product/platform!2679

pweyck committed:
Merge branch 'next-9899/remove-duplicate-condition' into '6.3.0.0'

NEXT-9899 - Remove duplicate condition

See merge request shopware/6/product/platform!2762

GitEvil committed:
NEXT-7521 - fix regressions

pweyck committed:
Merge branch 'next-7521/fix-snippet-regressions' into '6.3.0.0'

Next 7521/fix snippet regressions

See merge request shopware/6/product/platform!2765

shyim committed:
NEXT-9918 - Use theme asset package in default theme

OliverSkroblin committed:
Merge branch 'next-9918/use-theme-asset-package' into '6.3.0.0'

NEXT-9918 - Use theme asset package in default theme

See merge request shopware/6/product/platform!2769

OliverSkroblin committed:
Merge branch 'next-9886/remove-google-shopping-related-tables' into '6.3.0.0'

NEXT-9886 - Remove Google Shopping Related Tables

See merge request shopware/6/product/platform!2759

leichteckig committed:
NTR - Split validation e2e test into smaller specs

OliverSkroblin committed:
NEXT-9700 - Added possiblitiy to overwrite api generic routes

leichteckig committed:
NEXT-9957 - Add additional text to product slider

shyim committed:
NEXT-9767 - Deprecate StoreApiClient

taltholtmann committed:
NEXT-9962 - Fix tax rate sorting

Commit:
NEXT-9752 - Add fallback translation to installer and updater

pweyck committed:
Merge branch 'next-9700/api-overwrites' into '6.3.0.0'

NEXT-9700 - Added possiblitiy to overwrite api generic routes

See merge request shopware/6/product/platform!2781

OliverSkroblin committed:
Merge branch 'next-9767/deprecate-store-api-client' into '6.3.0.0'

NEXT-9767 - Deprecate StoreApiClient

See merge request shopware/6/product/platform!2784

taltholtmann committed:
Merge branch 'next-9962/fix-tax-rate-sorting' into '6.3.0.0'

NEXT-9962 - Fix tax rate sorting

See merge request shopware/6/product/platform!2788

pweyck committed:
Merge branch 'next-9752/add-fallback-translations-to-installer' into '6.3.0.0'

NEXT-9752 - Add fallback translation to installer/updater

See merge request shopware/6/product/platform!2789

GitEvil committed:
NEXT-7521 - fix snippets installer and storefront and tax module

pweyck committed:
Merge branch 'next-7521/general-snippet-fixes' into '6.3.0.0'

NEXT-7521 - fix snippets installer and storefront and tax module

See merge request shopware/6/product/platform!2791

pweyck committed:
Merge branch 'next-9957/add-additional-text-to-product-slider' into '6.3.0.0'

NEXT-9957 - add additional text to product slider

See merge request shopware/6/product/platform!2782

pweyck committed:
Merge branch 'ntr/create-e2e-validation-tests' into '6.3.0.0'

Ntr/create e2e validation tests

See merge request shopware/6/product/platform!2771

pweyck committed:
NEXT-9800 - Fix version in changelog

OliverSkroblin committed:
Merge branch 'ntr/fix-version-in-changelog' into '6.3.0.0'

NEXT-9800 - Fix version in changelog

See merge request shopware/6/product/platform!2809

Commit:
NEXT-10035 - Fix order confirmation mail template

pweyck committed:
Merge branch 'next-10035/fix-order-confirmation-mail-template-for-lineitems-without-options-payload-value' into '6.3.0.0'

NEXT-10035 - Fix order confirmation mail template

See merge request shopware/6/product/platform!2814

keulinho committed:
NEXT-7044 - Fix migration for deleted languages

pweyck committed:
Merge branch 'next-7044/fix-migration' into '6.3.0.0'

NEXT-7044 - Fix migration for deleted languages

See merge request shopware/6/product/platform!2835

jleifeld committed:
NEXT-10118 - fix user title when feature flag is not active

pweyck committed:
NEXT-10124 - Bump version

jleifeld committed:
NEXT-10118 - remove shopware testsuite cypress plugins

pweyck committed:
NEXT-10117 - Readd fallback for technicalName

pweyck committed:
Merge branch 'next-10118/user-title-fix' into '6.3.0.1'

NEXT-10118 - user title fix

See merge request shopware/6/product/platform!2843

pweyck committed:
Merge branch 'next-10117/readd-fallback-for-technical-name' into '6.3.0.1'

NEXT-10117/readd fallback for technical name

See merge request shopware/6/product/platform!2846

NTWEXO committed:
refactor listing filters ready for code review
